### PR TITLE
WIP 9.0 mig mail edit

### PR DESCRIPTION
**Pull request by gfcapalbo**
Mail_edit rewritten for 9.0: dan@sunflowerweb.nl  , tom@sunflowerweb.nl , gdaramouskas@therp.nl
hbrunn@therp.nl

### CODE REVIEW

**Comment by hbrunn** [comment]
read travis and runbot's output

**Comment by gfcapalbo** [comment]
thank you @ntsirintanis for the tests. ready for review [redacted]

**Comment by thomaspaulb** [comment]
@gfcapalbo If you want to ping us don't use our emailaddresses but our Github handles :-)

File: mail_edit/static/src/js/mail_edit.js [hunk 1]
```diff
@@ -1,50 +1,40 @@
...
-
-    instance.mail.MessageCommon.include({
-                init: function (parent, datasets, options) {
-                    this._super(parent, datasets, options);
-                    this.is_superuser = datasets.is_superuser || false;
-                }
+    thread.include({
+        init: function(){
+            this._super.apply(this, arguments);
+            this.events['click .o_edit'] = function (event) {
```
Comment (thomaspaulb) [inline mail_edit/static/src/js/mail_edit.js:11]: You ported the "edit" command, but not the "delete" command?

**Comment by hbrunn** [comment]
oh yes, I call shame! don't do that, most services protect mail addresses reasonably well by now, but you gave data to the scanners for giving them to people who leave me being annoyed. No reason to not still edit the comment to replace that by handles, then in the next run our mails won't be included at least from here

**Comment by thomaspaulb** [comment]
@gfcapalbo Could you still remove those emails?

**Review by thomaspaulb — APPROVED** [review]
We have a more advanced port for [12.0](https://github.com/OCA/social/pull/658), which covers 'delete' as well. We can merge this one for anyone that is still using 9.0.

**Pull request closed without merging**